cegs_portal/search/templates/search/v1/experiment.html

```diff
@@ -231,7 +234,7 @@
                     </div>
                     <div>
                         <label for="covSelect">Choose a Coverage View:</label>
-                        <select name="views" id="covSelect">
+                        <select class="search-button border-2 p-2 h-10 text-sm" name="views" id="covSelect">
```

Comment:
Given how many places it's used that aren't searching, I think it's time to rename the `search-button` class.